### ISSUE TITLE
Ability to add routines

### ISSUE THREAD

**Issue by srujayk** [qw3rtman/gg#5]
It would be cool to have the ability to make some of your own "routines". For example, if you had a system of adding all files, committing with a message, and then pushing the files, you could make a "routine" that allows you to do that all at once. 

Ex. 
![screen shot 2015-03-05 at 11 20 41 am](https://cloud.githubusercontent.com/assets/6854781/6510191/0dd2f8b2-c32a-11e4-9499-c217a92e3acc.png)

Instead of running those commands every time, you could make a routine, lets say using the command "rr" (stands for "run routine"), that would run all of those for you.

In other words, running the following command would add all files, commit using the message given, and push the files.

<pre>gg rr <i>’My Message’</i></pre>

**Comment by qw3rtman** [comment]
Currently, there is no API for adding custom commands, so the only way for you to do this right now is to add a command by hand.

This is a great idea and will be added to the TODO. Will try to implement this in the near future. :smile:

**Comment by srujayk** [comment]
Ok, thanks!
